packages/theme/README.md

```diff
@@ -35,7 +35,7 @@ Once a styled component is wrapped by the ThemeProvider, it can access
 the current theme from the `props.theme` property.
 
 Additionally, inside the theme object, you will find a `current` property
-that will be set to `light` or `dark` depending by the active theme.  
```

Comment:
Two trailing whitespaces are interpreted as newline in markdown, may you revert this change please?

Reply:
Hmm... is this whitespace different than space characters?

Reply:
No, they are just two white spaces. But markdown interprets 2 trailing white spaces this way.

Reply:
done